packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenu.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/OptionsMenu/options-menu.css';
+import { css, getModifier } from '@patternfly/react-styles';
+
+export enum OptionsMenuPosition {
+  right = 'right',
+  left = 'left'
+}
+
+export enum OptionsMenuDirection {
+  up = 'up',
+  down = 'down'
+}
+
+export interface OptionsMenuProps extends React.HTMLProps<HTMLDivElement>{
```

Comment:
it extends HTMLDivElement but I don't see the `<div>` below accepting `...props`

Reply:
Fixed!